content/json-ld/latest/events/event-status.md

```diff
@@ -0,0 +1,271 @@
+---
+---
+
+# Event - status
+
+The status indicates whether an event takes place as planned, has been postponed or cancelled. The following values for 'status' are applicable:
+* `available`: the event takes place as planned. This is the default value when no status for the event or subEvent is included.
+* `temporarilyUnavailable`: the event has been postponed to a later date yet to be determined.
+* `unavailable`: the event has been cancelled.
+
+Optionally, a reason for the new status can be added. 
+
+When the status is set back to `available`, the reasons (if present) will be automatically removed.
+
+It is possible to set the status on top event level or on subEvent level:
+* **top event level**: for the whole event and all its subEvents (occurences). All subEvents will inherit the status from the top event level
+* **subEvent level**: for a one or multiple subEvent(s). The status on top event level will automtically be derived from the (different) statusses on subEvent level.
+
+## Set status on top event level
+
+It is possible to set the status on top event level for all calendarTypes: `single`, `multiple`, `permanent`, `periodic`. If the event has subEvents, all subEvents of the event will inherit the status (and reason) from the top event level.
+
+## HTTP request
+
+```
+PUT /events/{eventId}/status
+```
+
+Update the status of the top event and optionally also the reason for the new status.
+
+
+### Request headers
+
+| Header        | Value            | Required? |
+| ------------- | ---------------- | --------- |
+| Authorization | Bearer {token}   | true      |
+| X-Api-Key     | {apiKey}         | true      |
+| Content-Type  | application/json | false     |
```

Comment:
Nit: Maybe we should just add some general "authentication" page and refer to that?
I also don't think we do anything with the `Content-Type` header, even if it's set. (Unless in some very specific cases maybe, but definitely not for status)

Reply:
(This was also added in other places.)

Reply:
True. The authorization part is repeated unnecessarily on this page (and other pages). 
I'll move the authentication info to the top of the page (for now). We can later on do a corrective action to delete all unnecessary Authorization documentation on individual pages and refer to the general Authentication page